instrumentation/google-http-client-1.19/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/googlehttpclient/AbstractGoogleHttpClientTest.java

```diff
@@ -83,7 +83,9 @@ public int sendRequest(HttpRequest request, String method, URI uri, Map<String,
       throws Exception {
     HttpResponse response = sendRequest(request);
     // read request body to avoid broken pipe errors on the server side
-    response.parseAsString();
+    if (!uri.toString().contains("/long-request")) {
+      response.parseAsString();
```

Comment:
can you add a comment why skipping this check for `/long-request`?

Reply:
Initially I used a longer delay that caused reading response to fail. Now this just makes the test a bit faster. I added a comment, but removing this `if` would also be fine.